alerting/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -5,4 +5,5 @@ grant {
 
     permission java.net.SocketPermission "*", "connect,resolve";
     permission java.net.NetPermission "getProxySelector";
+    permission java.lang.RuntimePermission "accessUserInformation";
```

Comment:
Why is this needed?

Reply:
This is temporary change to try to fix compile issue while rebasing to latest main branch